ui/helpers/utils/util.test.js

```diff
@@ -49,61 +49,61 @@ describe('util', () => {
     });
 
     it('should return true when given a valid subdomain', () => {
-      expect(util.isValidDomainName('foo.foo.bar')).toStrictEqual(true);
+      expect(util.isValidEnsDomainName('foo.foo.eth')).toStrictEqual(true);
     });
 
     it('should return true when given a single-character domain', () => {
-      expect(util.isValidDomainName('f.bar')).toStrictEqual(true);
+      expect(util.isValidEnsDomainName('f.eth')).toStrictEqual(true);
     });
 
     it('should return true when given a unicode TLD', () => {
-      expect(util.isValidDomainName('台灣.中国')).toStrictEqual(true);
+      expect(util.isValidEnsDomainName('台灣.eth')).toStrictEqual(true);
     });
 
     it('should return false when given a domain with unacceptable ASCII characters', () => {
-      expect(util.isValidDomainName('$.bar')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('$.eth')).toStrictEqual(false);
     });
 
     it('should return false when given a TLD that starts with a dash', () => {
-      expect(util.isValidDomainName('foo.-bar')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('foo.-eth')).toStrictEqual(false);
     });
 
     it('should return false when given a TLD that ends with a dash', () => {
-      expect(util.isValidDomainName('foo.bar-')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('foo.eth-')).toStrictEqual(false);
     });
 
     it('should return false when given a domain name with a chunk that starts with a dash', () => {
-      expect(util.isValidDomainName('-foo.bar')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('-foo.eth')).toStrictEqual(false);
     });
 
     it('should return false when given a domain name with a chunk that ends with a dash', () => {
-      expect(util.isValidDomainName('foo-.bar')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('foo-.eth')).toStrictEqual(false);
     });
 
     it('should return false when given a bare TLD', () => {
-      expect(util.isValidDomainName('bar')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('eth')).toStrictEqual(false);
     });
 
     it('should return false when given a domain that starts with a period', () => {
-      expect(util.isValidDomainName('.bar')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('.eth')).toStrictEqual(false);
     });
 
     it('should return false when given a subdomain that starts with a period', () => {
-      expect(util.isValidDomainName('.foo.bar')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('.foo.eth')).toStrictEqual(false);
     });
 
     it('should return false when given a domain that ends with a period', () => {
-      expect(util.isValidDomainName('bar.')).toStrictEqual(false);
+      expect(util.isValidEnsDomainName('eth.')).toStrictEqual(false);
     });
 
     it('should return false when given a 1-character TLD', () => {
-      expect(util.isValidDomainName('foo.b')).toStrictEqual(false);
```

Comment:
Changing test cases like this is clearly indicating the limiting effect the code changes will have on ENS resolution.